pom.xml

```diff
@@ -49,7 +49,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.10</version>
+            <version>4.11</version>
```

Comment:
Yes it is intentional, one of the tests was failing on the initial build of the library, this fixed it for me